src/bridge/test-packages.c

```diff
@@ -98,6 +97,12 @@ setup (TestCase *tc,
   if (fixture->expect)
     cockpit_expect_warning (fixture->expect);
 
+  if (fixture->lang)
+    {
+      g_setenv ("LANG", fixture->lang, TRUE);
+      g_setenv ("LC_MESSAGES", fixture->lang, TRUE);
+    }
```

Comment:
I figure the tests wouldn't need to set both, but it can't hurt either, so fine.